tests/load/k6-check-in-rush.js

```
@@ -0,0 +1,391 @@
+        timestamp: new Date().toISOString(),
+      };
+      expectedError = 'INVALID_QR_FORMAT';
+      break;
+      
+    case 1:
+      // Expired ticket
+      const expiredPayload = {
+        ticketId: 'EXPIRED_TICKET',
+        eventId: 'past_event',
+        expiryDate: new Date(Date.now() - 86400000).toISOString(),
+      };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)